src/components/map/Map.js

```diff
@@ -52,7 +56,19 @@ class Map extends React.Component {
                       .addTo(this.map)
                       .setZIndex(0);
   }
-
+    
+  updateLayers(layerUrls) {
```

Comment:
I still dont like this. this.map can easily be passed outside of the component to the Layers component and be added there. This is poluting the map component and adding complexity when it is there to be reusable. All of this code should be inside the component that manages the layers.